### PR TITLE
User can see articles based on their location 

### PR DESCRIPTION
**Pull request by Odeane**
#
### User Story
```
As a user
In order to read relevant articles close to my location
I would like to see relevant articles to be based on my location.
```
## Changes proposed in this pull request:
* User can view article based on location

## What I have learned working on this feature:
* How to retrieve longitudinal and latitudinal location
## Screenshots (Client)
Screenshot 2020-08-19 at 18 49 47

### CODE REVIEW

File: cypress/integration/visitorCanViewLocalNews.feature.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const stubLocation = require("../support/stubLocation");
+
```
Comment (Cma-B) [inline cypress/integration/visitorCanViewLocalNews.feature.js:2]: is that extra line? or should be like that?

File: cypress/integration/visitorCanViewLocalNews.feature.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+describe("visitors can veiw articles ", () => {
+  context('based on their based on sweden location', () => {
+    before(() => {
+      cy.server()
+      cy.route({
+        method: "GET",
+        url: "http://localhost:3000/api/v1/articles?latitude=60&longitude=18",
```
Comment (Cma-B) [inline cypress/integration/visitorCanViewLocalNews.feature.js:9]: just a question, why we use ? here at the end of url?

**Review by viktorsommar — APPROVED** [review]

File: cypress/integration/visitorCanViewLocalNews.feature.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      });
+    });
+
+    it("Visitors can see list article two", () => {
+      cy.get("#article-2").within(() => {
+        cy.get("#title").should("contain", "Sad Campers");
+        cy.get("#lead").should("contain", "Sad campers is always a losers");
+      });
+    });
+
```
Comment (Cma-B) [inline cypress/integration/visitorCanViewLocalNews.feature.js]: I guess this line is extra

**Review by Cma-B — COMMENTED** [review]
great job!

File: cypress/integration/visitorCanViewLocalNews.feature.js (hunk 2 above)
Comment (Odeane) [reply cypress/integration/visitorCanViewLocalNews.feature.js:9]: because of the query parameters that follow

**Review by oliverochman — APPROVED** [review]

**Pull request merged**